documentation/book/cluster-operator.adoc

```diff
@@ -947,6 +952,19 @@ INFO:: The Cluster Operator does not validate the provided configuration.
 When invalid configuration is provided, the Kafka Connect cluster might not start or might become unstable. 
 In such cases, the configuration in the `config` object should be fixed and the Cluster Operator will roll out the new configuration to all Kafka Connect instances.
 
+===== Logging
+The `logging` field allows the configuration of loggers. These loggers are:
+[source]
+log4j.rootLogger
+connect.root.logger.level
+log4j.logger.org.apache.zookeeper
+log4j.logger.org.I0Itec.zkclient
+log4j.logger.org.reflections
+
+To see options and examples how to set logging see <<logging_examples, logging examples>> for Kafka.
+
+When using external ConfigMap remember to place your custom ConfigMap under `log4j.properties` key.
```

Comment:
I don't think you need this, given that you have it in the examples section too.

Reply:
I kept it here because for Topic Operator it is `log4j2.properties`.

Reply:
Ah, of course! Well in that case we also need to note that the expected properties in that key are the log4j2 ones, rather than the log4j ones. Maybe we should link to external doc of properties.